CHANGELOG.md

```diff
@@ -1,6 +1,21 @@
 Changelog
 =========
 
+## 1.5.0 (2024-03-18)
+
+### Enhancements
```

Comment:
I think it would be good to include here a bit more detail about the deprecation.

How about:

```suggestion
### Deprecations

The `BugsnagPerformanceSwiftUI` is being deprecated and replaced with `BugsnagPerformanceSwift`, which will include all additional Swift code and not just SwiftUI. Existing users should change their imports to `BugsnagPerformanceSwift`. `BugsnagPerformanceSwiftUI` will no longer be updated after the next major release of this library.

### Enhancements
```